src/lib.rs

```diff
@@ -14,13 +14,14 @@ pub use crate::{
     browser::dom::cast::{
         to_html_el, to_input, to_kbevent, to_mouse_event, to_select, to_textarea,
     },
-    browser::service::fetch::{Method, Request},
+    browser::service::fetch::{Method, Request, ResponseDataResult, ResponseResult},
     browser::service::routing::push_route,
     browser::url::Url,
     browser::util::{
         self, body, canvas, canvas_context_2d, cookies, document, error, history, html_document,
         log, update, window,
     },
+    virtual_dom::{attrs::Attrs, listener::Listener, style::Style},
 };
```

Comment:
1. `virtual_dom::{Attrs, Listener, Style},` should be enough thanks to reimports.
2. If we need `Attrs` and `Style` I think it should be also added in the `prelude`.
3. How we'll decide what should be in `pub use crate` and what in `prelude`?

Reply:
What do you think about these specific things, on prelude vice crate? It's a balance between making things easy to get to, and not adding too many things to the global namespace. I think these are probably not common enough to be in the prelude; ie the main use I've found for them is in fn signatures.

Reply:
Well, with the common import `use seed::{prelude::*, *}` we can maybe remove `pub use crate::{`.
And I haven't used `Attrs` and `Style` at all in `kavik.cz` or `RealWorld ex`, only `Listener` in `window_events`.
I think we can wait once there are more Seed apps, then search through their code and change `prelude` / `pub crate` accordingly as the part of a bigger `lib.rs` refactor. It will be a breaking change but easily fixable one.